Вылетает трейс при ошибках в команде run

```
[*] picodata admin: ...
[*] picodata admin: 
Error: failed to execute Run command

Caused by:
    0: failed to enable plugins
    1: failed to execute picodata query ALTER PLUGIN "radix" MIGRATE TO 0.7.0;

Stack backtrace:
   0: anyhow::error::<impl anyhow::Error>::msg
   1: anyhow::__private::format_err
   2: cargo_pike::commands::run::enable_plugins
   3: cargo_pike::commands::run::cluster
   4: cargo_pike::commands::run::cmd
   5: cargo_pike::main
   6: std::sys::backtrace::__rust_begin_short_backtrace
   7: std::rt::lang_start::{{closure}}
   8: std::rt::lang_start_internal
   9: main
  10: __libc_start_call_main
             at ./csu/../sysdeps/nptl/libc_start_call_main.h:58:16
  11: __libc_start_main_impl
             at ./csu/../csu/libc-start.c:360:3
  12: _start
make: *** [Makefile:68: pico_start_release] Error 1
```

В нём нет смысла, надо скрыть

--- a/src/commands/run.rs
+++ b/src/commands/run.rs
@@ -722,7 +722,7 @@ pub fn cluster(params: &Params) -> Result<Vec<PicodataInstance>> {
                         error!("failed to kill picodata instances: {e:#}");
                     });
                 }
-                return Err(e.context("failed to enable plugins"));
+                bail!("failed to enable plugins: {}", e.to_string());
             }
         }
     };

--- a/tests/run.rs
+++ b/tests/run.rs
@@ -274,15 +274,11 @@ fn test_cluster_failure() {
     let cluster_status = run(&params);
     assert!(cluster_status.is_err(), "Expected migration error");
 
-    // Unwrap anyhow error chain and search for the error, containing MIGRATE keyword
     let err = cluster_status.unwrap_err();
-    let contains_migration_error = err
-        .chain()
-        .next()
-        .and_then(|e| e.source())
-        .is_some_and(|src| src.to_string().contains("MIGRATE"));
-
-    assert!(contains_migration_error);
+    assert!(
+        err.to_string().contains("MIGRATE"),
+        "expected migration error"
+    );
 }
 
 #[test]

